machine_configuration/100-intel-iommu-on.yaml

```diff
@@ -6,12 +6,13 @@ kind: MachineConfig
 metadata:
```

Comment:
Appreciated if yo can add a link with more detail description  about sm_on the commit log.

Reply:
added